bittide-instances/src/Bittide/Instances/Hitl/FullMeshHwCc.hs

```diff
@@ -266,32 +266,34 @@ fullMeshHwTest refClk sysClk IlaControl{syncRst = rst, ..} rxns rxps miso =
   capture = (captureFlag .&&. allUp) .||. unsafeToActiveHigh syncRst
 
   fincFdecIla :: Signal Basic125 ()
-  fincFdecIla = ila
-    (ilaConfig $
-         "trigger_0"
-      :> "capture_0"
-      :> "probe_milliseconds"
-      :> "probe_allStable0"
-      :> "probe_transceiversFailedAfterUp"
-      :> "probe_nFincs"
-      :> "probe_nFdecs"
-      :> "probe_net_nFincs"
-      :> Nil
-    ){depth = D16384}
-    sysClk
-
-    -- Trigger as soon as we come out of reset
-    (unsafeToActiveLow syncRst)
-
-    capture
-
-    -- Debug probes
-    milliseconds1
-    allStable0
-    transceiversFailedAfterUp
-    nFincs
-    nFdecs
-    (fmap unsignedToSigned nFincs - fmap unsignedToSigned nFdecs)
+  fincFdecIla =
+    setName @"fincFdecIla" $
+    ila
+      (ilaConfig $
+          "trigger_0"
```

Comment:
Some space seems to be lost during indentation here :upside_down_face:.

Reply:
I've removed the indentation I added as a whole